src/main/java/org/sciborgs1155/robot/Constants.java

```diff
@@ -36,10 +36,12 @@ public static enum RobotType {
   public static final double FULL_SPEED_MULTIPLIER = 1.0;
 
   public static class Field {
```

Comment:
look around for a team that has starting positions of every note and put them in here